### ISSUE TITLE
bullet list does not work when an image is on its left side

### ISSUE THREAD

**Issue by rjmaris** [splitbrain/dokuwiki#2245]
Because bullet lists don't work when text is floating around the right side of an image, I define images to have text floating around its left side. Then the bullets do display at all.

**Comment by rjmaris** [comment]
Problem does not occur on a modern web browser. I didn't do the usual "other browser test" when a problem occurs, because it seemed a coding issue. Indeed, Opera "presto engine" 12.16 makes the difference.

**Comment by rjmaris** [comment]
For sure, a buggy behaviour is present, whether it is in the css or in the code. The screenshot shows this (Firefox, Vivaldi):
![screenshot](https://user-images.githubusercontent.com/1149309/35874491-ed47a7f8-0b6c-11e8-85f6-62a2baa3f667.png) You can see that the (standard Dokuwiki) bullets do not appear indented, not even along the normal text margin, but a bit on the left side of this. In the preview (DW-editor), the bullets appear on the left of the image. When the image is set to appear on the right side, the bullets indent correctly.

**Comment by phy25** [comment]
Hi, do you mean the bullets before each line disappear (actually they are displaced on the left side of the image)? I can reproduce this on Chrome Windows once, but after I refreshed the page it disappeared 😕 

Test page: https://www.dokuwiki.org/playground:issue2245 (If you can reproduce this with your piece of DokuWiki syntax, please add there.)

As this cannot be reproduced all the time, I think this might be a browser rendering issue...

**Comment by rjmaris** [comment]
I have modified the contents of the test page.
In order to get an indication of the real problem area, I did the same experiment in a Wordpress website I run elsewhere: **The same behaviour**!

Curiously enough: apparently it occurs quite seldom that bullet lists are used together with images & text flow around images.

**Comment by rjmaris** [comment]
phy25, thank you very much for this, I added this into conf/userstyle.css and the bullets are represented correctly, even on opera 12.16.
Curiously, applying this on user defined wordpress css has another effect - bullets get simply invisible. Just for information.

**Comment by selfthinker** [comment]
This is an issue that is known for a long time. (See a [bug report in the old bug tracker](https://bugs.dokuwiki.org/1950.html) from 2010.) It is also an issue that you will see all over the internet (e.g. on Wikipedia) as it's a generalCSS problem that's hard/impossible to solve.

I am not aware of any way to fix this that doesn't have any side effects. We did fix it for a while with `overflow: hidden` but that introduced other bugs, so we removed it again and declared it closed with "won't fix".
